ansible/roles/universal_forwarder/tasks/collect_powershell_logs.yml

```diff
@@ -0,0 +1,11 @@
+- name: Create folder directory for inputs configuration
+  win_file:
+    path: "{{ item }}"
+    state: directory
+  with_items:
+    - 'C:\Program Files\SplunkUniversalForwarder\etc\apps\powershell_inputs_app\local'
```

Comment:
Do we need an app for each of these, why can't it be placed under their respective apps?

Reply:
yes because we don't use the full-blown TAs on the Universal Forwarder. We only use the needed inputs.conf.